awx/sso/pipeline.py

```diff
@@ -138,3 +143,73 @@ def update_user_teams(backend, details, user=None, *args, **kwargs):
         users_expr = team_opts.get('users', None)
         remove = bool(team_opts.get('remove', True))
         _update_m2m_from_expression(user, team.member_role.members, users_expr, remove)
+
+
+def update_user_orgs_by_saml_attr(backend, details, user=None, *args, **kwargs):
+    if not user:
+        return
+    from awx.main.models import Organization
+    from django.conf import settings
+    multiple_orgs = feature_enabled('multiple_organizations')
+    org_map = settings.SOCIAL_AUTH_SAML_ORGANIZATION_ATTR
+    if org_map.get('saml_attr') is None:
```

Comment:
You check for this, but you also allow the `saml_attr` field to be optional:

`fields.CharField(required=False, allow_null=True)`

Is there a reason to not just make this field required?

Reply:
Making the field required would result in a validation error for an empty dict, whereas `required=False` allows `{}` to be used in addition to `null` to indicate no configuration.